code/game/objects/items/holosign_creator.dm

```diff
@@ -86,24 +86,24 @@
 	name = "custodial holobarrier projector"
 	desc = "A holographic projector that creates hard light wet floor barriers."
 	holosign_type = /obj/structure/holosign/barrier/wetsign
-	creation_time = 20
+	creation_time = 0.6 SECONDS
 	max_signs = 12
 
 /obj/item/holosign_creator/security
 	name = "security holobarrier projector"
 	desc = "A holographic projector that creates holographic security barriers."
 	icon_state = "signmaker_sec"
 	holosign_type = /obj/structure/holosign/barrier
-	creation_time = 30
+	creation_time = 3 SECONDS
 	max_signs = 6
 
 /obj/item/holosign_creator/engineering
 	name = "engineering holobarrier projector"
 	desc = "A holographic projector that creates holographic engineering barriers."
 	icon_state = "signmaker_engi"
 	holosign_type = /obj/structure/holosign/barrier/engineering
-	creation_time = 30
-	max_signs = 6
+	creation_time = 0.6 SECONDS
+	max_signs = 12
```

Comment:
Do these block people or do anything special? I cant remember off the top of my head but 12 looks like a lot if they have any functionality. We have things like metal foam and smart foam for breaches too.

Reply:
They stop people on run mode. Doesn't do anything to seal breaches. Its purpose is to inform people that engineering maintenance is occuring in the area, and would be wise to not continue.

The area that needs maintenance can be quite large if there was an explosion or anything like that.